modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -238,6 +241,29 @@ export default class LargeVideoManager {
                 // If the intention of this switch is to show the avatar
                 // we need to make sure that the video is hidden
                 promise = container.hide();
+
+                if ((!shouldDisplayTileView(state) || participant?.pinned) // In theory the tile view may not be
+                // enabled yet when we auto pin the participant.
+
+                        && participant && !participant.local && !participant.isFakeParticipant) {
+                    // remote participant only
+                    const track = getTrackByMediaTypeAndParticipant(
+                        state['features/base/tracks'], MEDIA_TYPE.VIDEO, id);
+                    const isScreenSharing = track?.videoType === 'desktop';
```

Comment:
Can we use the `videoType` from `this.newStreamData` instead of trying to get the track info here again ?

Reply:
Theoretically we can! But we want to detect inconsistencies between desktop sharing icon (which uses exactly this code to check  if we do screen sharing or not) and the large video and by reusing the video type from `this.newStreamData` we can potentially miss an occurrence of the issue...

Reply:
ok makes sense.